privacyidea/static/components/token/views/token.enrolled.totp.html

```diff
@@ -1,9 +1,12 @@
 <div class="row">
-    <div class="col-sm-6" ng-hide="form['2stepinit'] && enrolledToken.rollout_state !== 'clientwait'">
+    <div ng-hide="form['2stepinit'] && enrolledToken.rollout_state !== 'clientwait'"
+         ng-class="$state.includes('token.wizard') == true && form['2stepinit'] == false ? 'col-sm-12' : 'col-sm-6'">
         <img width="{{ qrCodeWidth }}"
+             ng-class="$state.includes('token.wizard') == true && form['2stepinit'] == false ? 'centered' : 'not-centered'"
```

Comment:
If 2-step is activated, no centered button appears and the QR will not be centered.